internal/assert/assertions.go

```diff
@@ -0,0 +1,92 @@
+package assert
+
+// Testing check shortcuts
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+// Log message and fail if actualValue doesn't match expectedValue
+// *Note that DeepEqual will check the value pointed to by two pointers, ie don't use for checking
+//	pointer equality.
+func Equal(t *testing.T, expectedValue interface{}, actualValue interface{}, cm ...interface{}) bool {
+	if !reflect.DeepEqual(expectedValue, actualValue) {
+		msg := fmt.Sprintf("%v != %v (expected)", actualValue, expectedValue)
+		if len(cm) > 0 {
+			msg = msg + " - " + fmt.Sprintf(cm[0].(string), cm[1:]...)
```

Comment:
if the first value of `cm` is not a string, this will panic. you need to check the assertion or maybe change the function signature to accept a string then variadic interface{} params.